vyxal/main.py

```diff
@@ -143,7 +143,8 @@ def execute_vyxal(file_name, flags, inputs, output_var=None, online_mode=False):
 
     ctx.stacks.append(stack)
     try:
-        exec(code)
+        locals()["stack"] = stack
```

Comment:
assigning to `locals()[]` is technically undefined behaviour in Python

Reply:
Looks like it isn't needed anyway. Good catch.